scss/overrides/_type.scss

```diff
@@ -0,0 +1,18 @@
+a {
```

Comment:
If we are loading our foundation scss correctly, we shouldn't need these overrides for `a` here. Can you double check?

Reply:
Right, but I think we talked about this somewhere and can't find it 😅 I've tried to find out where this is coming from and from what I can see, it's not coming from any styles in this codebase, mofo-ui, or mofobootstrap. It's highhhh up on line 70 of our main.scss with other sitewide rules. Do you have an idea of where it may be coming from? 
![Screen Shot 2019-05-17 at 2 08 17 PM](https://user-images.githubusercontent.com/36484608/57947706-9b330c00-78ad-11e9-9ee4-f98e289434f5.png)

Reply:
p.s. the foundation-site doesn't have this issue because it doesn't have the same inital code in the compiled css.

Reply:
It's risky to add override for all `a` in this PR. Let's remove this entire chunk from this file and add the same rules to `footer.scss` targeting `.site-footer a {}` for now. I'll file a new ticket to track down where that `text-decoration: underline` rule for `a` is coming from.

Reply:
ticket filed: https://github.com/mozilla/network-pulse/issues/1072

Reply:
this rule is coming from `dist/css/main.css`...as far as if it's being compiled from elsewhere, not sure...but that's a starting point i guess.

Reply:
I'll update that ticket with the findings I discovered!